scripts/maven-release-publisher.yml

```
@@ -0,0 +1,37 @@
+version: 0.2
+env:
+  shell: /bin/sh
+  secrets-manager:
+    ORG_GRADLE_PROJECT_SONATYPE_NEXUS_USERNAME: amplify/android/sonatype:username
+    ORG_GRADLE_PROJECT_SONATYPE_NEXUS_PASSWORD: amplify/android/sonatype:password
+    ORG_GRADLE_PROJECT_signingPassword:  amplify/android/signing:password
+    ORG_GRADLE_PROJECT_signingKeyId:  amplify/android/signing:keyId
+    ORG_GRADLE_PROJECT_signingInMemoryKey: amplify/android/signing:inMemoryKey
+    ORG_GRADLE_PROJECT_signingSecretKeyRingFile:  amplify/android/signing:secretKeyRingFile
+phases:
+  install:
+    commands:
+      - echo 'Install phase starting'
+    finally:
+      - echo 'Install phase completed.'
+  pre_build:
+    commands:
+      - echo 'Pre-build phase starting'
+    finally:
+      - echo 'Pre-build phase completed.'
+  build:
+    commands:
+      - echo 'Build phase starting.'
+      - |
+        for item in aws-* core core-kotlin rxbindings; do 
+          task_name=":$item:uploadArchives"
```

```suggestion
        # List all available gradle tasks, grep for the uploadArchive tasks, and then use cut to strip the
        # task description and just return the name of the task, one for each module (e.g. aws-api:uploadArchives)
        for task_name in $(./gradlew tasks --all | grep uploadArchives | cut -d " " -f 1); do
```

Two benefits to this:
 - If we add/remove modules in the future, we won't have to update this script.
 - This logic can be used in any of our repos, not just `amplify-android`

---

I like it. I thought about just doing `./gradlew uploadArchives`, but I didn't test to see if there are any undesirable effects from doing that.